### PR TITLE
chore(travis): Add the ability to kick off deis e2e tests from travis

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-language: bash
+language: generic
```
Comment (sgoings) [inline .travis.yml:1]: Why the change?

Comment (jchauncey) [inline .travis.yml:1]: to match the logger yaml
